### ISSUE TITLE
an option to remove items from the queue when gestures are disabled

### ISSUE THREAD

**Issue by againstpetra** [UnicornsOnLSD/finamp#1566]
### Are you using the newest Version?

- [x] Yes, the feature is still missing on the newest version
- [x] No, there aren't any open request about this. I checked!

### Type of Feature

- [x] Something new
- [ ] Improvement of a feature
- [ ] None of the above

### Description

when gestures are disabled, there doesn't seem a way to delete items from the queue (usually this is done by swiping on an item in the queue)

### Reasoning

I always disable gestures, and this means I can't remove songs from a queue

### Implementation Notes

it should be pretty simple to do, I don't know the codebase though. an option being added under `Remove from this playlist` for the queue would probably work

<img width="1280" height="1180" alt="Image" src="https://github.com/user-attachments/assets/a47e6735-d9f1-492d-9579-70a48e55f69d" />

**Comment by Chaphasilor** [comment]
@againstpetra if gestures are disabled there *should* be an `x` button next to each track in the queue. Maybe we messed that up with a recent update?

Still, your suggestions seems sensible too. I'll try to add this for the next version.

**Comment by againstpetra** [comment]
this is what I see with gestures disabled: 

<img width="1280" height="1599" alt="Image" src="https://github.com/user-attachments/assets/5d70c68c-74fa-4d25-895a-9f6ee1d7ff5e" />

so yeah I think something's messed up if there's meant to be an `x` button :)
